### PR TITLE
Add media contacts to press centre

### LINKED ISSUE #242: Add 'Media Contacts' to press-centre

### PR DESCRIPTION
**Pull request by steverydz**
[Fixes #242](https://app.zenhub.com/workspace/o/canonical-websites/insights.ubuntu.com/issues/242)

download

### CODE REVIEW

File: templates/press-centre.html [hunk 1]
```diff
@@ -65,6 +65,39 @@ <h5 class="p-muted-heading">LinkedIn</h5>
...
+            </ul>
+          </div>
+          <div>
+            <h5 class="p-muted-heading">Cloud & IoT (US)</h5>
+            <ul class="p-inline-list--middot">
+              <li class="p-inline-list__item"><a href="mailto:Canonical@offleashpr.com">Canonical@offleashpr.com</a></li>
+            </ul>
+          </div>
+          <div>
+            <h5>All other enquiries</h5>
```
Comment (pmahnke) [inline templates/press-centre.html]: is this purposely not  `class="p-muted-heading"`?

**Comment by steverydz** [comment]
Yes, I copied the markup structure of the existing modules for consistency.
Happy to change if that would look better?

**Comment by pmahnke** [comment]
@frankiegranato what do you think?  I think it would look better all muted headings.

**Comment by steverydz** [comment]
With muted headings:
ubuntu insights

Without muted headings:
ubuntu insights 1

**Comment by steverydz** [comment]
Added class to final heading

**Comment by frankiegranato** [comment]
Looks great @steverydz 
One small request: Can we make all the email addresses start with a small letter please? So "canonical@berkeley.global
canonical@offleashpr.com"
(I'm sorry - this typo was probably caused by me in the original text I gave you!)

**Comment by steverydz** [comment]
@frankiegranato Done

ubuntu insights

**Comment by pmahnke** [comment]
Approving for @frankiegranato

**Pull request merged**